src/main/java/backend/resource/TurboIssue.java

```diff
@@ -413,6 +413,16 @@ public void markAsUnread(Preferences prefs) {
     }
 
     /**
+     * Checks if the labels assigned to this issue is the same as {@code labels}
+     * @param labels
+     * @return
+     */
+    public Boolean isLabelsSame(List<String> labels) {
+        List<String> currentLabels = this.getLabels();
+        return currentLabels.size() == labels.size() && currentLabels.containsAll(labels);
```

Comment:
I've been looking around for this. There is an `equals` method on `List` but the elements must be in the same order. Sorting or adding them to a set first are more complicated and less efficient provided the few number of labels on each issue. So this seems to be the simplest way I can think of so far when we don't have to worry about duplicates.